### ISSUE TITLE
720p HDTV added to backlog as low quality.

### ISSUE THREAD

**Issue by RadeoSuicide** [SickRage/sickrage-issues#423]
Branch: master
Commit: 4eccee2ad1b3e3d144bfd8e315fdf397cfd33361
Version: v5.1
Python 2.7.9 (default, Dec 10 2014, 12:28:03) [MSC v.1500 64 bit (AMD64)]
Windows 10

Hi, as of a recent update my SickRage has added all of my 720p HDTV episodes of my TV Shows to the backlog as low quality. This only affects 720p HDTV and not 720p WEB-DL or anything else. All of my TV Shows are SDTV as I don't mind lower quality videos.
Sorry for the lack of words in this post but I have trouble putting my thoughts into words :)

Thanks

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist)

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by labrys** [comment]
@RadeoSuicide This is likely due to the removal of Archive on First Match which changed how qualities are handled which was mentioned in SickRage's News.  Reading [Quality Changes](https://github.com/SickRage/sickrage-issues/wiki/Qualities-Changes) should allow you to update your qualities correctly.

**Comment by RadeoSuicide** [comment]
Thanks, how do I access the qualities options? I cant find it anywhere

**Comment by labrys** [comment]
Qualities can be set by editing a show's settings.  This can be done from a show's individual page by clicking the Edit button at the top, or can be done for a group of shows at once using the Mass Update page.

**Comment by labrys** [comment]
Low quality means its not in your settings.  If you want it to not show as low quality add 720p to the qualities.

**Comment by RadeoSuicide** [comment]
I have all of my shows set to SD quality so that SickRage only snatches SD torrents. 720p HDTV is higher quality than SDTV and SD DVD, which is what all of my shows are set to. Do I need to set all of the affected shows to 720p quality and then replace all of the 720p torrents that it snatches for them?

**Comment by RadeoSuicide** [comment]
Sorry about that, I didn't think that I would get a response.

**Comment by labrys** [comment]
As per the link on quality changes:

> If you have qualities in the Preferred list set, any episode matching a quality in this list will not be replaced with a better quality. AKA it is a "best" match and is final.

> If you ONLY have qualities in the Allowed list (Or have a preset selected other than custom) it will also stop on the first match in this list and be considered a "best" match, and never be replaced with a better quality.

> If you have qualities in BOTH lists, it will accept any quality in the Allowed list, and continue looking for a better quality in the Allowed list, until it finds one in the Preferred list.

##### Here are some configuration options for you as examples:
###### Option 1
Download SDTV, SD DVD,  or 720p HDTV
* Allowed qualities: SDTV, SD DVD, and 720p HDTV

This will try to download SDTV, SD DVD, or 720p HDTV.  If any of the above are downloaed, it will not replace them.

###### Option 2
Download SDTV, SD DVD,  or 720p HDTV but prefer SDTV and SD DVD. 
* Allowed qualities: SDTV, SD DVD, and 720p HDTV
* Preferred qualities: SDTV, SD DVD

This will try to download SDTV or SD DVD, if it can't find those it will download 720p HDTV instead.  The 720p HDTV will be replaced with an SDTV or SD DVD in the future if one is found.  720p will show up as LOW QUALITY since it is not one of your preferred qualities even though 720p is of higher quality.

**Comment by RadeoSuicide** [comment]
I will do that then. With the more popular shows will the regular HDTV torrents be chosen over the higher quality copies do to the preferred quality being SD and SD DVD?

**Comment by RadeoSuicide** [comment]
I have set all of my shows to 
Allowed: SDTV, SD DVD, 720p HDTV, 1080p HDTV, 720p WEB-DL, 1080p WEB-DL, 720p BluRay, 1080p BluRay
Preferred: SDTV, SD DVD, 720p HDTV, 1080p HDTV, 720p WEB-DL, 1080p WEB-DL, 720p BluRay, 1080p BluRay
and then added 720p and 1080p to the not allowed words list. 
Instead of showing any episode of not preferred quality as being low quality, shouldn't it just show episodes of actual lower quality as being low quality? You know what I mean? Like if I have a show has 720p as the quality and I have a couple of 1080p episodes of it, SR shouldn't care unless I tell it to. Only lower quality episodes like SD should be marked as being low quality.

**Comment by labrys** [comment]
@RadeoSuicide No problem about the other thread :).  Just wanted you to know why it was closed.  It also helps people follow an issue more closely when its contained to a single thread.

**Comment by labrys** [comment]
@RadeoSuicide As for the "Low Quality" it's a poorly worded label.  Non-preferred Quality is more descriptive of the actual issue but is a little wordy.  An improved quality system is in the works and this is one of the issues it will address.

**Comment by RadeoSuicide** [comment]
Ok thanks a lot for all of your help :)

**Comment by rcocchiararo** [comment]
I just noticed i had an isuse with backlog trying to download higher/lower qualities... it does not seem to work like the help says.

I have set all my series to 720p (not custom). For some reason, i ended up with some 1080p episodes (sickrage was not finding a 1080p one and i manually got it).

The backlog still shows those, and they are shown in yellow in the serie's list.

I manually updated all my series to 1080p and then 720, and nothing changed.

Now i have set those series with the issue to custom, selecting 720 and 1080 in both allowed and accepted, and it is gone from the backlog and displayed in green.

PS: i have one particular anime that was set to 1080p, and i manually got a 720p episode cause i sickrage (or me) could not find it in 1080p.
i had to manually set custom as explained before to get it accepted (as in, 720p and 1080p in both preffered and allowed).
